---a/net/minecraft/block/BlockHalfStoneSlab.java
+++b/net/minecraft/block/BlockHalfStoneSlab.java
@@ -1,12 +1,11 @@
 package net.minecraft.block;
 
-public class BlockHalfStoneSlab extends BlockStoneSlab
-{
+public class BlockHalfStoneSlab extends BlockStoneSlab {
+
     private static final String __OBFID = "CL_00002108";
 
-    public boolean isDouble()
-    {
+    public boolean isDouble() {
         return false;
     }
-}
+}
